packages/lib/server/queries/teams/index.ts

```diff
@@ -15,19 +15,16 @@ export async function getTeamWithMembers(args: {
   slug?: string;
   userId?: number;
   orgSlug?: string | null;
+  isTeamView?: boolean;
+  isOrgView?: boolean;
```

Comment:
Added the flexibility we need to optimize returned object to improve page size on certain usages